src/App.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import { useState } from 'react';
+
+//const chats = chatMessages;
+
+function App() {
+  const [chatData, setChatData] = useState(chatMessages);
+
+  // const calculateTotalLikes = (chatData) => {
+  //   const totalLikes = 0;
+  //   chatData.map((entry) => {
+  //     totalLikes += entry.liked;
+  //   });
+  //   return totalLikes;
+  // };
```

Comment:
This is really, really close to what is needed to calculate the number of likes. Remember that `entry.liked` is a boolean, not a number. You cannot add a boolean to `totalLikes`. Instead, you should 
```
if (entry.liked) {
    totalLikes += 1;
}
```
That would get you the total number of likes.